src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -156,6 +156,8 @@ public class HelixWatch3DViewModel : DefaultWatch3DViewModel
         private readonly Color4 defaultSelectionColor = new Color4(new Color3(0, 158.0f / 255.0f, 1.0f));
         private readonly Color4 defaultMaterialColor = new Color4(new Color3(1.0f, 1.0f, 1.0f));
         private readonly Color4 defaultTransparencyColor = new Color4(1.0f, 1.0f, 1.0f, 0.5f);
+        private readonly Color4 defaultIsolatedTransparencyColor = new Color4(1.0f, 1.0f, 1.0f, 0.3f);
```

Comment:
Are these all defined by Dynamo team somehow?

Reply:
@mjkkirschner where did you get the `0.3f` from? Is it specified somewhere?

Reply:
not that I could find, I just tried to match the existing look, where isolated geometry is more transparent than frozen geometry.

Reply:
Looks like this value was originally set to 0.1f for lines and points: https://github.com/DynamoDS/Dynamo/pull/7587/files#diff-20b13dfe2f88aaf8df37e1240b79c201R30 and 0.2f .

Reply:
@aparajit-pratap - thanks for finding this!